src/python/WMCore/WMBS/File.py

```diff
@@ -587,6 +587,7 @@ def addFilesToWMBSInBulk(filesetId, workflowName, files, isDBS=True,
                            wmbsFile["first_event"],
                            wmbsFile['merged']])
 
+    files = []  # explicitly delete it
```

Comment:
This doesn't delete the files parameter, if you are intending to do that

Reply:
do you mean, the files parameter with in the method? when we pass the parameter to the function parameter reference is copied so even if you do "files = []" in the function, original (outside function variable "file") will still point to whole list. So it won't delete the list.

If you really want to clean the memory you need to do,

"del files[0:]" instead of "files = []".

But I won't do it.
1. It is dangerous do this thing in the method. (maybe it will be use outside the method) If not it should still be deleted outside method.
2. Even if we delete the reference, we don't control garbage collection schedule.  So it is not guarantee that memory will be free right away. 
we can do gc.collect() to trigger garbage collection if this is really necessary.

Reply:
i will actually remove it